src/Citeproc/Eval.hs

```diff
@@ -1142,16 +1148,64 @@ evalItem layout (position, item) = do
            , contextPosition = position
            },
         st{ stateReference = ref
-          , stateUsedYearSuffix = False }))
+          , stateUsedYearSuffix = False
+          , stateUsedIdentifier = False
+          , stateUsedTitle = False
+          }))
         $ do xs <- mconcat <$> mapM eElement (layoutElements layout)
+
+             -- find identifiers that can be used to hyperlink the title 
+             let mbident = 
+                    foldl (<|>) Nothing
+                      [ IdentDOI   <$> (valToText =<< lookupVariable "DOI" ref)
+                      , IdentPMCID <$> (valToText =<< lookupVariable "PMCID" ref)
+                      , IdentPMID  <$> (valToText =<< lookupVariable "PMID" ref)
+                      , IdentURL   <$> (valToText =<< lookupVariable "URL" ref)
+                      ]
+             let mburl = identifierToURL <$> mbident
+            
+             -- hyperlink any titles in the output
+             let linkTitle url (Tagged TagTitle x) = Linked LinkTitle url [x]
+                 linkTitle _ x = x
+             
+            -- ensure correct handling of link prefixes like (https://doi.org/)
+            -- when a link's prefix+anchor=target, ensure the link includes the prefix
+            -- (see pandoc#6723 and citeproc#88; also pandoc's fixLinks function)
```

Comment:
Presumably we'll remove fixLinks from pandoc after this is added here, so referring to it in the comment may be confusing.

Reply:
Sure, I removed it.